Makefile

```diff
@@ -1,48 +1,29 @@
-GITHUB_API_TOKEN := ""
-
-##
 # Site
-site-fetch-contributors:
-	GITHUB_API_TOKEN=$(GITHUB_API_TOKEN) node ./site/scripts/fetch-contributors.js
```

Comment:
Were there any obstacles that didn't let you keep fetching contributors by script?

Reply:
No there wasn't. I just changed the data structure of the contributors from YAML to JSON, and added it back to the repo :)